docs/docker.md

```diff
@@ -1,45 +1,78 @@
-# Linux Headless Agent
+# Docker Linux Headless Agent
 
-To run the agent, simply specify a few environment variables with docker:
+The `Dokerfile` has multi-stage definition:
+* **production**: Default stage, produce a image without debug features;
+* **debug**: When running the produced image the wptagent script will wait for a debug to.attach
+
+## Build the Image
 
-* `SERVER_URL`: will be passed as `--server` (note: it must end with '/work/')
-* `LOCATION`: will be passed as `--location`
-* `KEY`: will be passed as `--key`
-* `NAME`: will be passed as `--name` (optional)
-* `SHAPER`: will be passed as `--shaper` (optional)
-* `EXTRA_ARGS`: extra command-line options that will be passed through verbatim (optional)
+Arguments can be passed at build time:
+* **TIMEZONE**: to set the timezone inside the container
```

Comment:
Is `TIMEZONE` the only argument available at build time? Can you add sample values?

Maybe have the environment variables up here or before the arguments.

Reply:
there was also `DEBIAN_FRONTEND` but I removed it since I don't see any value in allowing interaction during installation at container build time.

I'll add an example.